tests/common.go

```diff
@@ -5138,6 +5142,7 @@ func ParseFlags() {
 	flag.StringVar(&pdsDriverName, pdsDriveCliFlag, defaultPdsDriver, "Name of the pdsdriver to use")
 	flag.StringVar(&anthosWsNodeIp, anthosWsNodeIpCliFlag, "", "Anthos admin work station node IP")
 	flag.StringVar(&anthosInstPath, anthosInstPathCliFlag, "", "Anthos config path where all conf files present")
+	flag.BoolVar(&skipSystemChecks, skipSystemCheckCliFlag, false, "Skip system check")
```

Comment:
Before merge, add more description to what this flag means by providing couple of actual checks that get skipped.